tests/test_api_comments.py

```diff
@@ -1,20 +1,11 @@
 import json
-import typing
 from typing import Any, Dict, List
 
 import pytest
 import responses
 
-from tests.data.test_defaults import (
-    DEFAULT_REQUEST_ID,
-    INVALID_ENTITY_ID,
-    REST_API_BASE_URL,
-)
-from tests.utils.test_utils import (
-    assert_auth_header,
-    assert_id_validation,
```

Comment:
I can see this being removed everywhere, but not from test_utils. Is this still needed?

Reply:
Thanks, forgot to remove this. Resolved by: https://github.com/Doist/todoist-api-python/pull/61/commits/aee3b29e862da7fa50b270bd5392162deed48190